docs/docs/getting-started.md

```diff
@@ -4,82 +4,82 @@ title: Getting Started
 permalink: docs/getting-started.html
 next: tutorial.html
 redirect_from: "docs/index.html"
+redirect_from: "downloads.html"
 ---
 
-## JSFiddle
+There are several different ways to use React. The right one for you depends on how you're using it.
 
-The easiest way to start hacking on React is using the following JSFiddle Hello World examples:
+## Trying Out React
 
- * **[React JSFiddle](https://jsfiddle.net/reactjs/69z2wepo/)**
- * [React JSFiddle without JSX](https://jsfiddle.net/reactjs/5vjqabv3/)
+If you're just interested in playing around with React, you can use JSFiddle. Try starting from [this Hello World example code](https://jsfiddle.net/o9gspf3e/). You don't need to install anything; you can just modify the code and click "Run" to see if it works.
 
-## Create React App
+## Creating a New Application
```

Comment:
I would say `Creating a Single Page Application` because this is the specific use case CRA handles. For anything else, you should just use the bundler and do things your way.